SwiftUserDefaults.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+  s.name = "SwiftUserDefaults"
+  s.version = "0.0.1"
+  s.summary = "A series of Swift friendly utilities for Foundation's UserDefaults class"
+  s.homepage = "https://github.com/cookpad/swift-user-defaults"
+  s.license = { :type => "MIT", :file => "LICENSE.txt" }
+  s.author = { 'liamnichols' => 'liam.nichols.ln@gmail.com' }
+  s.source = { :git => "https://github.com/cookpad/swift-user-defaults.git", :tag => "v#{s.version}" }
+  s.source_files = "Sources/**/*.{swift}"
+  s.swift_version = "5.3"
+
+  # iOS deployment target needs to be at minimum iOS 14 so SwiftUI library references compile during `pod lib lint`.
+  s.ios.deployment_target = '14.0'
```

Comment:
Running `bundle exec pod lib lint --verbose --no-clean` allows us to debug this a bit further since it provides us with the console output and the project that it generated.

The error is strange though:

> Cannot find type 'AppStorage' in scope

Because SwiftUI should be available in Xcode 12 and 13 so the symbols should always be visible when we try to compile.

I dug up further through the logs and saw that this is the command that fails:

> ```bash
> $ /usr/bin/xcodebuild clean build 
>     -workspace /var/[...]/App.xcworkspace
>     -scheme App 
>     -configuration Release 
>     CODE_SIGN_IDENTITY=- 
>     -sdk iphonesimulator 
>     -destination id=217A8997-4224-4BF4-8DB9-7E08921CD0C9
> ```

The difference between running elsewhere is that this is trying to build in the **release** configuration. We can actually reproduce the issue in Xcode by opening Package.swift and trying to build after modifying the scheme to use the Release configuration for the "Run" action (and then pressing ⌘ + B)

Now I don't understand why this fixes it, but problem seems to be specifically with using a deployment target of iOS 10 only. Changing it to iOS 11 seems to work. 

Given that iOS 10 is 5 years old, we can probably just drop support altogether so lets change this here:

```suggestion
  s.ios.deployment_target = '11.0'
```

Please can you also update **Package.swift** to change it to a minimum of iOS 11 :bow:

Good job that we will run `pod lib lint` since this would have caught us out later on when we tried to integrate the pod 😅